sway-lib-std/src/low_level_call.sw

```diff
@@ -0,0 +1,105 @@
+library low_level_call;
+
+use ::assert::assert;
+use ::bytes::Bytes;
+use ::contract_id::ContractId;
+use ::option::Option;
+use ::revert::require;
+use ::vec::Vec;
+
+pub struct CallParams {
+    coins: u64,
+    asset_id: ContractId,
+    gas: u64,
+}
+
+// TODO : Replace with `from` when implemented
+fn contract_id_to_bytes(contract_id: ContractId) -> Bytes {
+    let mut target_bytes = Bytes::with_capacity(32);
+    target_bytes.len = 32;
+
+    __addr_of(contract_id).copy_bytes_to(target_bytes.buf.ptr, 32);
+
+    target_bytes
+}
+
+/// Represent a raw pointer as a Bytes
```

Comment:
Perhaps an additional note describing this data structure is in order? It still seems a bit like a staircase designed by M.C Escher even though we know it works :)

Reply:
Resolved [`73ff24c`](https://github.com/FuelLabs/sway/pull/3626/commits/73ff24bc9612bca45298c80d90236f8146efa55f)